Cargo.toml

```diff
@@ -2,7 +2,7 @@
 members = ["contracts/*", "packages/*"]
 
 [workspace.package]
-version = "0.4.18"
+version = "0.4.19"
```

Comment:
I have bumped this just to have an higher version and test everithing. Let me know if I should change it.